db/tasks.php

```diff
@@ -42,6 +33,15 @@
         'day' => '*',
         'dayofweek' => '*',
         'month' => '*'
+    ),
+    array(
+        'classname' => 'tool_crawler\task\crawl_manager',
+        'blocking' => 0,
+        'minute' => '00',
+        'hour' => '1',
```

Comment:
this hour => 1 looks wrong

Moot if we do not rename this class

Reply:
I've removed changes here to this scheduled task. We're just using the old one